install/common/pagespeed.conf.template

```diff
@@ -321,15 +321,19 @@
     # and change server state, such as statistics, caches, and
     # messages.  This might be appropriate in an experimental setup.
     <Location /pagespeed_admin>
-        Order allow,deny
-        Allow from localhost
-        Allow from 127.0.0.1
+        Require all granted
+        Require local
+        #Require ip myremoteip
```

Comment:
Do we need these commented lines? If so, can you add a comment as to document why we need them? :-)